pyspark/test/simple_integration_test.py

```diff
@@ -122,5 +122,16 @@ def test_predict(self):
         for i in range(0, total_length):
             self.assertTrue(np.allclose(p[i], ground_label[i], atol=1e-6, rtol=0))
 
+    def test_rng(self):
+        rng = RNG()
+        rng.set_seed(100)
+        result = rng.uniform(0.1, 0.2, [2, 3])
```

Comment:
Check if the result still the same if ``` uniform``` being invoked multiple times with the same seed.